### DISCUSSION TITLE
opam repository

### DISCUSSION THREAD

**Discussion by gAlleb** [savonet/liquidsoap#3391] [Installation] [answered]
Hello! I know that liquidsoap 2.2.1 has been released few days ago. When it will be available via opam upgrade?
As I understand it takes time for opam to update its public repository?
Thanks.

**Comment by smimram** [comment, marked as answer]
Yes, the corresponding PR to update the packages can be tracked there: https://github.com/ocaml/opam-repository/pull/24376.

**Reply by gAlleb** [reply]
Thank you!
